modules/button/react/stories/stories_Button.tsx

```diff
@@ -86,6 +86,10 @@ storiesOf('Components/Buttons/Button/React/Standard', module)
           Primary
         </Button>
       </div>
+      <h3>Mirrored Primary</h3>
```

Comment:
Do you mind reverting your additions to all of the button stories and adding them to our visual testing stories instead?  ([see here](https://workday.github.io/canvas-kit/?path=/story/testing-react-buttons-button-button--button-states))

In [modules/button/react/stories/visual-testing/stories_Button.tsx](https://github.com/Workday/canvas-kit/blob/master/modules/button/react/stories/visual-testing/stories_Button.tsx#L33):
```tsx
        mirrorIcon: [
          {value: undefined, label: ''},
          {value: true, label: 'Mirrored Icon'},
        ],
```

You'll probably also want to make use of the [filter function](https://github.com/Workday/canvas-kit/blob/master/utils/storybook/ComponentStatesTable.tsx#L63) accepted by `permutateProps` so that this only adds rows when icon is defined:
```tsx
({icon, mirrorIcon}) => {
  if (mirrorIcon && !icon) {
    return false
  }
}
```